main.cpp

```diff
@@ -1,95 +1,570 @@
+5 properties:
+    1)sales volume
+    2)release year
+    3)number of songs
+    4)length of album  
+    5)price of album
+3 things it can do:
+    1)save the image of cover
+    2)read the brief introduction of a album    
+    3)play a song in album
  */
-
+struct Album
+{
+    //1)sales volume
+    int salesVolume = 130023;
+    //2)release year
+    int releaseYear = 1998;
+    //3)number of songs
+    int numberOfSongs = 10;
+    //4)length of album
+    float lengthOfAlbum = 32.5f;   
+    //5)price of album
+    float priceOfAlbum = 34.99f;
+    //1)save the image of cover
+    void saveTheImageOfCover(char addressOfCover);
+    //2)read the brief introduction of a album
+    void ReadTheBriefIntroductionOfAAlbum(char briefIntroduction);
+    //3)play a song in album
+    void playASongInAlbum (char addressOfSong);
+};
 /*
- 8)
+8) Ranking 
+5 properties:
+    1)ranking of a song
+    2)number of ranking changes of a song
+    3)period of the ranking     
+    4)peak
+    5)the number of people who share the ranking to social platform
+3 things it can do:
+    1)play the music of a ranking
+    2)check a ranking of this week
+    3)Favorite a ranking
  */
-
+struct Ranking
+{
+    //1)ranking of a song
+    int rankingOfASong = 17;
+    //2)number of ranking changes of a song
+    int numberOfRankingChangesOfASong = -2;
+    //3)period of the ranking
+    int periodOfTheRanking= 7;
+    //4)peak
+    int peak = 3;
+    //5)number of people who share the ranking to social platform
+    int numberOfPeoplaWhoShareTheRankingToSocialPlatform = 466;
+    //1)play the music of a ranking
+    void playTheMusicOfARanking(char nameOfASong);
+    //2)check a ranking of this week
+    void checkARankingOfThisWeek(int numberOfWeek);
+    //3)Favorite a ranking
+    void favoriteARanking(int numberOfPlaylist);
+};
 /*
- 9)
+9) Music interview
+5 properties:
+    1)host
+    2)artist    
+    3)number of word
+    4)date
+    5)topic
+3 things it can do:
+    1)Read the interview
+    2)comment on the interview
+    3)watch the video of the interview
  */
-
+struct MusicInterview
+{
+    //1)host
+    std::string host = "Samy";
+    //2)artist
+    std::string artist = "Lana Del Rey";    
+    //3)number of word
+    int numberOfWord = 1979;
+    //4)date
+    int date = 23;
+    //5)topic
+    std::string topic = "Who is Lana Del Rey";
+    //1)Read the interview
+    void readTheInterview(char addressOfInterview);
+    //2)comment on the interview
+    void commentOnTheInterview(int numberOfComment);
+    //3)watch the video of the interview
+    void watchTheVideoOfTheInterview(char addressOfVideoOfTheInterview);
+};
 /*
- 10)
+10）Music app
+5 properties:
+    1)User
+    2)Artist
+    3)Album
+    4)Ranking
+    5)Music interview
+3 things it can do
+    1) listen to music
+    2) subscribe membership
+    3) make comment certain piece of music
  */
-
+struct MusicApp
+{
+    //1)User
+    struct User
+    {
+    //1)user registration days
```

Comment:
Do not copy the `User` struct you declared on line 287 and paste it inside this class.
Give your class an INSTANCE of the `User` struct:
```
struct MusicApp
{
    //1)User
    User user; //this declares an instance of `User` inside of the `MusicApp. 
    //it's the same concept as declaring a `float int bool double char` member variable inside of a UDT.
```
Do this for the 4 other properties you defined in plain-english for this class
- [ ] revise